airbyte-integrations/connectors/source-airtable/source_airtable/streams.py

```diff
@@ -157,7 +157,7 @@ def request_params(self, next_page_token: Mapping[str, Any] = None, **kwargs) ->
         """
         All available params: https://airtable.com/developers/web/api/list-records#query
         """
-        params = {}
+        params = {"pageSize": 5}
```

Comment:
this will be removed in the final review. I just set this value so that I can test the checkpointing behavior and restarting